bubblewrap.c

```diff
@@ -1327,6 +1333,10 @@ main (int    argc,
   if (!is_privileged)
     opt_unshare_user = TRUE;
 
+  if (opt_unshare_user_try &&
+      stat ("/proc/self/ns/user", &sbuf) == 0)
```

Comment:
Okay, so you're arguing RHEL is the outlier?  I find that hard to
argue with, but to me a major benefit of bubblewrap is that it can
target RHEL (as of today).

Anyways...not strictly objecting to this patch but it also seems like
something one could easily do outside of bubblewrap as I said.